openslides/utils/cache.py

```diff
@@ -77,7 +77,8 @@ def __init__(
 
         # Start time is used as first change_id if there is non in redis
         if start_time is None:
-            start_time = int((datetime.utcnow() - datetime(1970, 1, 1)).total_seconds())
+            # Use the miliseconds (rounted) since the 2016-02-29.
```

Comment:
Nice. I like this date ...

Reply:
Was something else there together with the leap day?
I would suggest taking the 2011-9-12

Reply:
It has its reason.